x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/enrich/EnrichPolicyResolver.java

```diff
@@ -98,81 +96,111 @@ public record UnresolvedPolicy(String name, Enrich.Mode mode) {
     /**
      * Resolves a set of enrich policies
      *
-     * @param targetClusters     the target clusters
+     * @param targetClusters     the target clusters; key: cluster alias; value: skip_unavailable setting (null for local cluster)
```

Comment:
Looks like local cluster key is removed anyway, why have a special value for it? Can't we just use e.g. `false`? In fact, maybe `resolvePolicies` should have `includeLocal` flag instead too? Otherwise we are creating a map to only copy it and create another map which is the same but without local. This map is usually small but still looks strange.

Reply:
The comment is wrong (old). I removed the `(null for local cluster)` part of the comment.

The code was originally structured this way to pass in a Set and remove the local, setting a boolean. Now it passes in a Map and removes the local. I could avoid the copying step, but it's generally bad practice to modify most incoming params in case some other usage of this method ever happens.

If we change it to not remove the local, it requires several additional downstream pieces of code, so I just left the model in place.